src/Models/PdbColumn.php

```diff
@@ -74,4 +74,74 @@ public function check(array $tables, PdbTable $table): array
 
         return $errors;
     }
+
+
+    /**
+     * Gets the first key value pair of an iterable
+     *
+     * @param iterable $iter An array or Traversable
+     * @return array|null An array of [key, value] or null if the iterable is empty
+     * @example
+     *          list ($key, $value) = Sprout::iterableFirst(['an' => 'array']);
+     */
+    public static function iterableFirst($iter)
+    {
+        foreach ($iter as $k => $v) {
+            return [$k, $v];
+        }
+
+        return null;
+    }
```

Comment:
This exists in the kb Arrays helper. Really don't want to start polluting these models with this kind of stuff.